Ontology/Space/Room/HospitalRoom/HospitalRoom.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:building:HospitalRoom;1",
```

Comment:
Recommend changing to **HealthcareRoom**

Reply:
Agreed -- will fix and update PR. May take a day or two though.

Reply:
Now fixed in latest update to this PR.